dev/build_prevalidation.py

```diff
@@ -87,12 +87,9 @@ def main():
   init_argument_parser(parser)
   options = parser.parse_args()
 
-  annotator = Annotator(options)
-  halyard_bump = __annotate_component(annotator, 'halyard')
-
   bom_generator = BomGenerator(options)
   bom_generator.determine_and_tag_versions()
-  # bom_generator.determine_and_tag_halyard()
+  halyard_bump = bom_generator.determine_and_tag_halyard()
```

Comment:
You can get rid of the `__annotate_component` function, I believe it's not used anywhere else.